addons/binding/org.openhab.binding.atlona/README.md

```diff
@@ -179,44 +177,47 @@ The # of presets allowed depends on the firmware you are using (5 presets up to
 As of firmware 1.6.03 (rev 13), there are three issues on Atlona firmware (I've notified them on these issues):
 
 -   clearX command does not work.  The TCP/IP command "ClearX" as specified in Atlona's protocol will ALWAYS return a "Command Failed".  Please avoid this channel until atlona releases a new firmware.
--   There is no way to query what the current status is of: panellock, and irenable.  This addon simply assumes that panellock is off and irenable is on at startup.  
--   If you make a change in the switches UI that requires a reboot (mainly changing any of the settings on the "Network" tab in the switch configuration UI), this addon's connection will be inconsistently closed at different times.  The thing will go OFFLINE and then back ONLINE when the reconnect attempt is made - and then it starts all over again.  Please make sure you reboot as soon as possible when the switch UI notifies you.
+
+-   There is no way to query what the current status is of: panellock, and irenable.  This addon simply assumes that panellock is off and irenable is on at startup.
```

Comment:
Consider changing to, "There is no way to query the current status of: panellock and..."

Also consider moving the next sentence starting, "This addon simply..." to its own line.